src/components/forms/dashboard/links/csv-shorten.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
+
+import Papa from "papaparse";
+import { toast } from "sonner";
+
+import { quickLinkShorten } from "@/actions/link-actions";
+import { Button } from "@/components/ui/button";
+
+import { useTransition } from "react";
+
+const CsvShortenForm = () => {
+  const [loading, startTransition] = useTransition();
+  interface CSVRow {
+    links: string;
+  }
+  function isURL(str: string): boolean {
+    const urlRegex = /^(?:https?|ftp):\/\/(?:\w+\.?)+/i;
+    return urlRegex.test(str);
+  }
+  const handleCSV = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+    if (file) {
+      console.log(file.type);
+      if (file.type === "text/csv") {
```

Comment:
Can we show an error toast if the filetype is not a CSV?

`toast.error("Please upload a CSV file")`